uwrt_mars_rover_hw/drivers/roboteq_driver/src/example_main.cpp

```diff
@@ -0,0 +1,164 @@
+#include <cassert>
```

Comment:
should delete this

Reply:
I deleted this. It's helpful for testing small changes like execptions tho

Reply:
Might be worth starting to integrate unit tests into this repo using gtest and rostest. http://wiki.ros.org/Quality/Tutorials/UnitTesting . i'll start the effort sometime in the coming weeks maybe